src/drone_controller/exception/exceptions.py

```diff
@@ -0,0 +1,15 @@
+class DroneControllerError(Exception):
+    """
+    Base class for exception in this module.
+    """
+
+
+class UserInputError(DroneControllerError):
+    """
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Multi-line docstring summary should start at the first line](https://app.codacy.com/gh/treeandsea/DroneController/pullRequest?prid=5205179)